starknet_py/tests/e2e/account/outside_execution_test.py

```diff
@@ -52,70 +49,99 @@ async def test_account_outside_execution_any_caller(
         caller=ANY_CALLER,
     )
 
-    tx = await argent_account.execute_v1(calls=[call], max_fee=MAX_FEE)
-    await argent_account.client.wait_for_tx(tx.transaction_hash)
+    tx = await account.execute_v3(
+        calls=[call],
+        l1_resource_bounds=ResourceBounds(
+            max_amount=int(1e5), max_price_per_unit=int(1e13)
+        ),
+    )
+    await account.client.wait_for_tx(tx.transaction_hash)
 
 
 @pytest.mark.asyncio
 async def test_account_outside_execution_for_invalid_caller(
-    argent_account: BaseAccount,
+    argent_account_v040: BaseAccount,
     account: BaseAccount,
     map_contract,
 ):
-    assert any(
-        [
-            await argent_account.supports_interface(OutsideExecutionInterfaceID.V1),
-            await argent_account.supports_interface(OutsideExecutionInterfaceID.V2),
-        ]
-    )
+    random_address = 0x1234567890123456789012345678901234567890
 
     put_call = Call(
         to_addr=map_contract.address,
         selector=get_selector_from_name("put"),
         calldata=[20, 20],
     )
 
-    call = await argent_account.sign_outside_execution_call(
+    call = await argent_account_v040.sign_outside_execution_call(
         calls=[
             put_call,
         ],
         execution_time_bounds=OutsideExecutionTimeBounds(
             execute_after=datetime.datetime.now() - datetime.timedelta(hours=1),
             execute_before=datetime.datetime.now() + datetime.timedelta(hours=1),
         ),
-        caller=account.address,
+        caller=random_address,
     )
 
-    tx = await argent_account.execute_v1(calls=[call], max_fee=MAX_FEE)
+    tx = await account.execute_v3(
+        calls=[call],
+        l1_resource_bounds=ResourceBounds(
+            max_amount=int(1e5), max_price_per_unit=int(1e13)
+        ),
```

Comment:
nit: We can use [`MAX_RESOURCE_BOUNDS_L1`](https://github.com/software-mansion/starknet.py/blob/39af414389984efbc6edc48b0fe1f914ea5b9a77/starknet_py/tests/e2e/fixtures/constants.py#L51) (same for other occurrences).
```suggestion
        l1_resource_bounds=MAX_RESOURCE_BOUNDS_L1,
```

Reply:
Thank you! It's good to know.

Reply:
Should be fine now